src/components/ButtonPanel.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+class ButtonPanel extends React.Component {
+  render() {
+    return (
+      <div>
+        <div className="group-1">
+          <button type="button">AC</button>
+          <button type="button">+/-</button>
+          <button type="button">%</button>
+          <button type="button">÷</button>
```

Comment:
Hi, let's use the button component we made here, passing the buttonName as a  prop.